harness/determined/deploy/local/cluster_utils.py

```diff
@@ -112,7 +112,7 @@ def get_proxy_addr() -> str:
 
 def _wait_for_master(master_host: str, master_port: int, cluster_name: str) -> None:
     try:
-        wait_for_master(master_host, master_port, timeout=100)
```

Comment:
is removing the timeout meant to be permanent?  If so, why?

Reply:
not needed, since if you look into the code, 100 is the default anyway.

Reply:
good example of why I prefer avoiding defaults in internal functions... explicit args just make everything so darn obvious.